packages/web3-react/src/components/Pages/Connectors/index.tsx

```diff
@@ -34,29 +34,14 @@ import {
 
 import { isMobile } from '../../../utils'
 import useDefaultWallets from '../../../wallets/useDefaultWallets'
+import { WalletProps } from '../../../wallets/wallet'
+import { routes } from '../../Common/Modal'
 
-const Wallets: React.FC = () => {
-  const context = useContext()
+const Connectors: React.FC = () => {
+  const context = useAlephiumConnectContext()
   const mobile = isMobile()
   const wallets = useDefaultWallets()
 
-  const findInjectedConnectorInfo = (name: string) => {
-    let walletList = name.split(/[(),]+/)
-    walletList.shift() // remove "Injected" from array
-    walletList = walletList.map((x) => x.trim())
-
-    const hasWalletLogo = walletList.filter((x) => {
-      const a = wallets.map((wallet: any) => wallet.name).includes(x)
-      if (a) return x
-      return null
-    })
-    if (hasWalletLogo.length === 0) return null
-
-    const foundInjector = wallets.filter((wallet: any) => wallet.installed && wallet.name === hasWalletLogo[0])[0]
-
-    return foundInjector
-  }
-
```

Comment:
I moved this function outside of the component so that it's easier to read it and so that it doesn't get redefined every time this component renders :)